web_poet/overrides.py

```diff
@@ -166,8 +200,8 @@ def wrapper(cls):
 
         return wrapper
 
-    def get_overrides(self) -> List[OverrideRule]:
-        """Returns all of the :class:`~.OverrideRule` that were declared using
+    def get_overrides(self) -> List[ApplyRule]:
+        """Returns all of the :class:`~.ApplyRule` that were declared using
```

Comment:
@kmike , in line with `OverrideRule` becoming `ApplyRule`, I was thinking if we should also rename the following:
- `get_overrides` to `get_rules`
- `search_overrides` to `search_rules`

I think "rules" is a more apt terminology with the introduction of the "to_return" since it means that "overrides/instead_of" can now be set to `None`. For example, `ApplyRule(for_patterns=Patterns(include=['example.com'], exclude=[], priority=500), use=CustomProductPage, to_return=Product)` loses the concept of overrides.

Do you see some possible divergence of the concepts if we do rename them?

Reply:
This makes sense 👍